autogpt_platform/backend/backend/api/features/onboarding_dump/brain_dump_gate_eval.py

```
@@ -0,0 +1,194 @@
+    eval error: in production that request ends in a non-personalized
+    ``failed`` status, which is the same protection the gate provides —
+    but it is still recorded, because "the STT errored" and "the gate
+    caught a hallucination" are different findings.
+    """
+    transcription_error: str | None = None
+    try:
+        result = await transcription.transcribe(
+            audio_path.read_bytes(), audio_path.name
+        )
+        transcript = result.text
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)